extensions/ql-vscode/src/databaseFetcher.ts

```diff
@@ -409,23 +418,31 @@ function convertRawLgtmSlug(maybeSlug: string): string | undefined {
   return;
 }
 
+function parseLgtmUrl(lgtmUrl: string): string | undefined {
+  const re = new RegExp('https://lgtm.com/projects/(g|gl|b|git)/(.*)');
```

Comment:
Only project slugs that begin with a `g` are from github (they are the vast majority of projects on lgtm. It may not be safe to match on other slugs. Eg- if two people put up the different projects with the same project name with different capitalization on github and bitbucket, you would get a mismatch. It's a rare case, but it's probably safer to protect against it.

Reply:
Also, please add a comment that this only supports github slugs.